featurebyte/api/event_view.py

```diff
@@ -152,7 +152,7 @@ def _getitem_frame_params(self) -> dict[str, Any]:
     def get_join_column(self) -> str:
         # This is potentially none for backwards compatibility.
         # We can remove this once DEV-556 is done.
-        assert self.event_id_column is not None
+        assert self.event_id_column is not None, "Event ID column is not available."
         return self.event_id_column
```

Comment:
This assertion could cause error when subsetting the view of table with ID column is `None`:
https://github.com/featurebyte/featurebyte/blob/e57a800dacc9532cef550672ab7ca1b81894a0fa/featurebyte/api/view.py#L949-L962

Reply:
The assertion already exists now - I only changed the error message here. Should we revise the function to be able to return `None` instead?